sandrinodimattia/gitbook-express-auth0-boilerplate
# Feature/es6 refactor

Hey, I've refactored the code base to use Babel and direnv environment variables.

To test

- check out the branch "git checkout feature/es6-refactor"
- set up the environment variables
- run `npm i` to install and build
- run each node script and navigate to the docs to ensure they are there.
